net-news/alligator/alligator-23.04.49.9999.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_TEST="forceoptional"
+KFMIN=5.75.0
+QTMIN=5.15.0
+inherit ecm gear.kde.org
+
+DESCRIPTION="A convergent RSS/Atom feed reader for Plasma"
+HOMEPAGE="https://apps.kde.org/alligator/"
+
+LICENSE="|| ( GPL-2 GPL-3 ) CC0-1.0 CC-BY-SA-4.0 GPL-3+"
+SLOT="0"
+KEYWORDS=""
+
+DEPEND="
+	>=dev-libs/kirigami-addons-0.6:5
+	>=dev-qt/qtdeclarative-${QTMIN}:5
+	>=dev-qt/qtgui-${QTMIN}:5
+	>=dev-qt/qtnetwork-${QTMIN}:5
+	>=dev-qt/qtquickcontrols-${QTMIN}:5
```

Comment:
I don't see evidence of this being used, not in CMakeLists.txt or qml files.

Reply:
It's reported by iwdevtools

Reply:
could you paste the output here please?

Reply:
lddtree output: https://paste.debian.net/1281264/

Reply:
oops, some of them are indeed unused

Reply:
Removed `dev-qt/qtquickcontrols` from deps

Reply:
yes, again only meant to comment on `dev-qt/qtquickcontrols` in particular